docs/content/deployment/overview.mdx

```diff
@@ -13,52 +24,254 @@ width={2986}
-| [Dagit](/concepts/dagit/dagit)                                                | Dagit serves the user interface and responds to GraphQL queries. It can have 1 or more replicas.                                               | Yes                                                                                |
-| [Dagster Daemon](/deployment/dagster-daemon)                                  | The Dagster Daemon operates schedules, sensors, and run queuing. Currently, replicas are not supported.                                        | Required for schedules, sensors, and run queuing                                   |
-| [User Repository Deployments](/concepts/repositories-workspaces/repositories) | User Repository Deployments serve metadata about user jobs. You can have many repository servers, and each server can have 1 or more replicas. | Recommended. Otherwise, Dagit and the Daemon must have direct access to user code. |
+**For larger deployments**, you can run and manage the code location servers separately.
 
-### Configuring Dagster Deployments
+---
 
-Dagster deployments are composed of multiple components, such as storages, executors, and run launchers. One of the core features of Dagster is that each of these components is swappable and configurable.
+## Deployment configuration
 
-If you don't provide any custom configuration, Dagster automatically uses a default implementation of each component. However, you can swap out the default implementation for another Dagster-provided one or even write a custom implementation. For example, Dagster uses the <PyObject module="dagster._core.storage.runs" object="SqliteRunStorage" /> by default to store information about pipeline runs. You can swap it out with the Dagster-provided <PyObject module="dagster_postgres" object="PostgresRunStorage"/> instead or even write your own custom storage class.
+Dagster OSS deployments are composed of multiple components, such as storages, executors, and run launchers. One of the core features of Dagster is that each of these components is swappable and configurable.
 
-Based on the component's scope, the component is configured either at the Dagster Instance or Job Run level. Access to user code is configured at the Workspace level. The table below describes how to configure components at each of these levels.
+If custom configuration isn't provided, Dagster will automatically use a default implementation of each component. However, default implementations can be swapped for another built-in Dagster option or your own custom implementation. For example, by default Dagster uses <PyObject module="dagster._core.storage.runs" object="SqliteRunStorage" /> to store information about pipeline runs. This can be swapped with the Dagster-provided <PyObject module="dagster_postgres" object="PostgresRunStorage"/> instead or or a custom storage class.
```

Comment:
The last sentence from the previous paragraph can be merged with this paragraph. And then, this sentence can be deleted: "However, default implementations can be swapped for another built-in Dagster option or your own custom implementation"